i6_models/primitives/specaugment.py

```diff
@@ -1,3 +1,4 @@
+import numpy as np
```

Comment:
We mostly use `import numpy` in other code (consistent to `import torch`), here also in the repo (although not too much yet), but also in RETURNN.
Although this variant here is more the official recommended way.

So maybe for i6_models, we can independently decide now what variant to pick and then always keep it consistent.

Reply:
You merged without resolving this here, or commenting here. So what do you mean?

Reply:
Ah now I see #38.

Reply:
As this is an independent non-critical issue, I would not delay this PR by that. If we decide different to this PR we can just change it back.